pylsp_rope/plugin.py

```diff
@@ -34,7 +39,7 @@ def pylsp_settings():
             # "pylint_lint": {"enabled": False},
             # "references": {"enabled": False},
             # "rope_completion": {"enabled": False},
-            # "rope_rename": {"enabled": False},
+            "rope_rename": {"enabled": True},
```

Comment:
This should be disabled by default and use a different config key to prevent any interaction with the base pylsp rope_rename and jedi_rename.